storybook/stories/CenterView/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { View } from 'react-native'
-import style from './style'
+import styles from './style'
 import DefaultImageBackground from '../../../src/common/ImageBackground'
 
-export default function CenterView ({ children }) {
+export default function CenterView ({ children, style }) {
   return (
-    <DefaultImageBackground>
-      <View style={style.main}>{children}</View>
+    <DefaultImageBackground style={style}>
```

Comment:
DefaultImageBackground must not use external styles. It is kind of constant component.